src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,36 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { User, UserInfo } from '../UserInfo';
+
+export type Todo = {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+  user: User | null,
+};
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    id, title, completed, user,
+  } = todo;
+
+  return (
+    <>
```

Comment:
'<>' you can use this tag when your component return multiple HTML tags, but in this case component return only one one tag article, so you can remove  this '<>'